amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -152,9 +152,14 @@ vx_status Rotate_Register(vx_context);
 vx_status Saturation_Register(vx_context);
 vx_status SequenceRearrange_Register(vx_context);
 vx_status Snow_Register(vx_context);
+vx_status Pixelate_Register(vx_context);
```

Comment:
Why is pixelate added in this PR. Please remove this

Reply:
Done